? Diff_that_fixes_None_encryption.diff
? underflow_test.pl
? lib/Palm/.Keyring.pm.swp
Index: lib/Palm/Keyring.pm
===================================================================
RCS file: /cvs/scripts/misc/Palm/lib/Palm/Keyring.pm,v
retrieving revision 1.51
diff -u -r1.51 Keyring.pm
--- lib/Palm/Keyring.pm	13 Sep 2007 15:46:09 -0000	1.51
+++ lib/Palm/Keyring.pm	2 Dec 2007 03:05:54 -0000
@@ -306,7 +306,8 @@
         }
         my $packed = _pack_field($field);
 
-        $rec->{data} = join $EMPTY, $packed, $rec->{ivec}, $rec->{encrypted};
+        $rec->{data} = join $EMPTY, 
+            $packed, $rec->{ivec}, $rec->{encrypted}, chr(0xff) x 2;
 
     } else {
         croak "Unsupported Version $self->{version}";
